programs/client/Client.cpp

```diff
@@ -1234,6 +1236,160 @@ void Client::processConfig()
     client_info.quota_key = config().getString("quota_key", "");
 }
 
+void Client::readArguments(
```

Comment:
@taiyang-li could you please explain, why we extracted this method as ClientBase abstraction ?

Reply:
Same reason as above.